src/main/java/io/candydoc/ddd/extract_domain_vocabulary/ReflectionsDomainVocabularyFinder.java

```diff
@@ -161,10 +163,27 @@ private static Set<FullName> dependenciesOf(Class<?> domainModel, Set<Class<?>>
     return dependenciesOfTacticalPattern(domainModel);
   }
 
+  private static List<Class<?>> getFieldType(Field field) {
+    Type fieldType = field.getGenericType();
+    if (fieldType instanceof ParameterizedType) {
+      List<Class<?>> actualArgumentClasses = new ArrayList<>();
+      Type[] actualTypeArguments = ((ParameterizedType) fieldType).getActualTypeArguments();
+      for (Type type : actualTypeArguments) {
+        if (type instanceof Class) {
```

Comment:
1) how can it be different than class ?

2) What about parameterized types in deeper levels ?

```java
Map<Integer, Map<String, Map<ValueObject,List<SomeEntity>>>
```

Reply:
Answer to 1.)